jobs/models.py

```diff
@@ -69,7 +68,81 @@ def set_expiration_date(self):
                 self.expiration_date = self.published_date + timedelta(60)
                 self.save()
 
-
     def __unicode__(self):
         return "{0}, {1}".format(self.title, self.company)
 
+
+class Meetup(models.Model):
+
+    MEETUP = 'MEET'
+    CONFERENCE = 'CONF'
+    WORKSHOP = 'WORK'
+    MEETUP_TYPES = (
+        (MEETUP, 'meetup'),
+        (CONFERENCE, 'conference'),
+        (WORKSHOP, 'workshop'),
+    )
+
+    title = models.CharField(max_length=500)
+    organization = models.ForeignKey(
+        'Company',
+        related_name="meetups",
+        blank=True,
+        null=True,
+    )
+    type = models.CharField(max_length=4, choices=MEETUP_TYPES, default=MEETUP)
+    contact_email = models.EmailField(max_length=254)
+    city = models.CharField(max_length=100)
+    country = CountryField()
+    description = models.TextField(max_length=5000)
+    is_recurring = models.BooleanField(
+        default=False,
+        help_text="Is your meetup recurring?"
+    )
+    meetup_date = models.DateTimeField(
+        null = True,
+        help_text="This stands for a starting date if the meetup is recurring"
+    )
+    reviewer = models.ForeignKey(
```

Comment:
Could be a good idea. I quess, right now we somehow assume that the person will write the recurrence type in the description field. But what if they forget..

I'll make this field optional. I guess it would be the best if that field was required when the is_recurring fields is true. Shall I do it in the model or shall we add it to the form validation?